### DISCUSSION TITLE
Localization - Generate POT : Can we call POT generate throght Godot CLI？Or glob Support？

### DISCUSSION THREAD

**Discussion by kaluluosi** [godotengine/godot-proposals#6567] [Editor]
Generate POT is a wonderfull feature, but it's not friendly for ci/cd pipline. 
I mean we need a CLI command to auto update POT.
And we need it support glob.
Like This:
![image](https://user-images.githubusercontent.com/1620585/227728842-1f32e6a5-7fe5-4034-8d50-ab343fe840bb.png)

**Comment by Calinou** [comment]
It should be feasible to add a command line argument to call the POT generator (similar to command-line export).

Regarding glob support, one difficulty is that you're expected to choose existing file paths using EditorFileDialog right now. A glob is not an existing file path, so we'd need to signal it in the UI somehow that you can also use globs. (Godot also lacks support for `**` in globs right now, and `*` will match every character including subfolders.)

**Comment by and-rad** [comment]
How about using something like the export UI for selecting files for POT generation?

![export](https://user-images.githubusercontent.com/8700280/228450021-0eac2e80-647f-4cf4-8d2d-5780178086eb.PNG)

It seems to me that half the functionality is already present here, so why not build on that? It could even be refactored into its own high-level widget for whenever a number of assets need to be selected from the project directory.

**Reply by kaluluosi** [reply]
It looks good.

**Comment by dalexeev** [comment]
There is a proposal about this. See:

* #10986
* https://github.com/godotengine/godot/pull/77584
* https://github.com/godotengine/godot/pull/98422

Thanks for the contribution nonetheless!
